crates/holochain/src/conductor/handle.rs

```diff
@@ -894,31 +902,75 @@ impl ConductorHandleT for ConductorHandleImpl {
     async fn create_clone_cell(
         self: Arc<Self>,
         payload: CreateCloneCellPayload,
-    ) -> ConductorResult<CellId> {
+    ) -> ConductorResult<InstalledCell> {
         let CreateCloneCellPayload {
-            properties,
-            dna_hash,
-            installed_app_id,
-            agent_key,
+            app_id,
             role_id,
+            properties,
+            network_seed,
             membrane_proof,
+            origin_time,
+            name,
```

Comment:
I wonder if we should have a `DnaPhenotypeOption` with Options for the 3 fields. Maybe overkill, but could be helpful to make sure these 3 properties are grouped.

Reply:
I agree to introduce it.